lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -26,7 +26,8 @@ module Framework
           tracer: Datadog.tracer,
           debug: false,
           trace_agent_hostname: Datadog::Writer::HOSTNAME,
-          trace_agent_port: Datadog::Writer::PORT
+          trace_agent_port: Datadog::Writer::PORT,
+          env: ::Rails.env
```

Comment:
For what it's worth, `env: nil` in my initializer brings back previous behavior (`env:` tag applied by the DD agent config).